### PR TITLE
Scores calculation feature

### PR DESCRIPTION
**Pull request by HeemahMuslad**

### CODE REVIEW

File: README.md
Comment (iamwatchdogs) [inline README.md:1]: Please remove the changes done within the `README.md` file. Each PR should created for a single purpose.

**Review by iamwatchdogs — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: README.md [hunk 1]
```diff
@@ -19,7 +19,8 @@
 
 </div>
 
-Welcome to **Javascript-Projects**, one of the best ways to introduce the world of open-source contribution! Whether you're a beginner looking to take your first steps or an intermediate developer seeking to refine your skills, this repository is designed to make your journey into open-source development. This repo will provide with best hands-on understanding of open source contributions.
+Welcome to **Javascript-Projects**, one of the best ways to introduce the world of open-source contribution! Whether you're a beginner looking to take your first steps or an intermediate developer seeking to refine your skills, this repository is designed to make your journey into open-source development. This repo will provide you with best hands-on understanding of open source contributions.
+
```
Comment (iamwatchdogs) [inline README.md:23]: Please remove the new line on line no.23 of the `README.md` file.
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -34,7 +35,7 @@ Hey there, fellow developer !!!... I'm happy to see you are interested in contri
 > [!IMPORTANT]
 > Make sure to check the [CONTRIBUTING.md](https://github.com/Grow-with-Open-Source/Javascript-Projects/blob/main/CONTRIBUTING.md "goto CONTRIBUTING.md") to understand the rules, file structure and step-by-step guide for contribution.
 
-- **Step 1:** Make sure you have required tools within you local machine *(like *git, vs code, node.js and so on)*.
+- **Step 1:** Make sure you have the required tools within you local machine *(like *git, vs code, node.js and so on)*.
```
Comment (iamwatchdogs) [inline README.md:38]: Please remove the _the_ from the sentence from line no.38 of the `README.md` file.
```suggestion
- **Step 1:** Make sure you have required tools within you local machine *(like *git, vs code, node.js and so on)*.
```

**Review by iamwatchdogs — CHANGES_REQUESTED** [review]
You have removed previously made changes but not all of them. Please make the requested changes.

**Comment by HeemahMuslad** [comment]
I have deleted the branchSent from my Huawei phone-------- Original message --------From: Shamith ***@***.***>Date: Tue, 31 Oct 2023, 3:17 pmTo: Grow-with-Open-Source/Javascript-Projects ***@***.***>Cc: Rohimat Mustapha ***@***.***>, Assign ***@***.***>Subject: Re: [Grow-with-Open-Source/Javascript-Projects] Scores calculation feature (PR #22)
@iamwatchdogs requested changes on this pull request.

You have removed previously made changes but not all of them. Please make the requested changes.

In README.md:
> @@ -19,7 +19,8 @@
 
 </div>
 
-Welcome to **Javascript-Projects**, one of the best ways to introduce the world of open-source contribution! Whether you're a beginner looking to take your first steps or an intermediate developer seeking to refine your skills, this repository is designed to make your journey into open-source development. This repo will provide with best hands-on understanding of open source contributions.
+Welcome to **Javascript-Projects**, one of the best ways to introduce the world of open-source contribution! Whether you're a beginner looking to take your first steps or an intermediate developer seeking to refine your skills, this repository is designed to make your journey into open-source development. This repo will provide you with best hands-on understanding of open source contributions.
+

Please remove the new line on line no.23 of the README.md file.
⬇️ Suggested change
-





In README.md:
> @@ -34,7 +35,7 @@ Hey there, fellow developer !!!... I'm happy to see you are interested in contri
 > [!IMPORTANT]
 > Make sure to check the [CONTRIBUTING.md](https://github.com/Grow-with-Open-Source/Javascript-Projects/blob/main/CONTRIBUTING.md "goto CONTRIBUTING.md") to understand the rules, file structure and step-by-step guide for contribution.
 
-- **Step 1:** Make sure you have required tools within you local machine *(like *git, vs code, node.js and so on)*.
+- **Step 1:** Make sure you have the required tools within you local machine *(like *git, vs code, node.js and so on)*.

Please remove the the from the sentence from line no.38 of the README.md file.
⬇️ Suggested change
-- **Step 1:** Make sure you have the required tools within you local machine *(like *git, vs code, node.js and so on)*.
+- **Step 1:** Make sure you have required tools within you local machine *(like *git, vs code, node.js and so on)*.

**Comment by iamwatchdogs** [comment]
@HeemahMuslad Please make the requested changes and please text something more understandable.

File: README.md [hunk 3]
```diff
@@ -19,8 +19,7 @@
 
 </div>
 
-Welcome to **Javascript-Projects**, one of the best ways to introduce the world of open-source contribution! Whether you're a beginner looking to take your first steps or an intermediate developer seeking to refine your skills, this repository is designed to make your journey into open-source development. This repo will provide with best hands-on understanding of open source contributions.
-
+Welcome to **Javascript-Projects**, one of the best ways to introduce the world of open-source contribution! Whether you're a beginner looking to take your first steps or an intermediate developer seeking to refine your skills, this repository is designed to make your journey into open-source development. This repo will provide you with best hands-on understanding of open source contributions.
```
Comment (iamwatchdogs) [inline README.md:22]: Please update the previous content.

```suggestion
Welcome to **Javascript-Projects**, one of the best ways to introduce the world of open-source contribution! Whether you're a beginner looking to take your first steps or an intermediate developer seeking to refine your skills, this repository is designed to make your journey into open-source development. This repo will provide with best hands-on understanding of open source contributions.

```

**Review by iamwatchdogs — CHANGES_REQUESTED** [review]

**Pull request closed without merging**